### PR TITLE
🚸 add pros::IMU alias

### LINKED ISSUE #288: Add Alias for pros::IMU as pros::Imu
Issue is what it says on label, the rationale behind this is because newer users get confused with the API or even mispellings of the fully uppercase IMU type.

### PR DESCRIPTION
**Pull request by AyushShukla3**
#### Summary:
Create the pros::Imu alias for pros::IMU in imu.h and imu.hpp

#### Motivation:
Reduce confusion with newer users attempting to use pros::Imu instead of pros;:IMU

##### References (optional):
closes #288

### CODE REVIEW

File: include/api.h [hunk 1]
```diff
@@ -40,9 +40,9 @@
 #endif /* __cplusplus */
 
 #define PROS_VERSION_MAJOR 3
-#define PROS_VERSION_MINOR 4
-#define PROS_VERSION_PATCH 0
-#define PROS_VERSION_STRING "3.4.0"
+#define PROS_VERSION_MINOR 3
+#define PROS_VERSION_PATCH 2
+#define PROS_VERSION_STRING "3.3.2-dirty.10.5abe157"
```
Comment (HotelCalifornia) [inline include/api.h:45]: Don't check this file into features or bugfixes

File: include/pros/imu.h [hunk 2]
```diff
@@ -28,6 +28,8 @@ namespace pros {
 namespace c {
 #endif
 
+typedef struct IMU Imu;
```
Comment (HotelCalifornia) [inline include/pros/imu.h:31]: does this even compile? there are no such structs defined in the C API. I think it is sufficient to provide that alias for the C++ API only.

File: version [hunk 3]
```diff
@@ -1 +1 @@
-3.4.0
+3.3.2-dirty.10.5abe157
```
Comment (HotelCalifornia) [inline version:1]: this file should also not be checked into changes like this

**Review by HotelCalifornia — CHANGES_REQUESTED** [review]

**Comment by HotelCalifornia** [comment]
also if you edit the description above to say "closes #288" it will automatically close that issue when this is merged

**Review by WillXuCodes — CHANGES_REQUESTED** [review]
Newline on Version

File: include/pros/imu.hpp [hunk 4]
```diff
@@ -444,6 +444,9 @@ class Imu {
 	 */
 	virtual bool is_calibrating() const;
 };
+
+	using IMU = Imu;
```
Comment (HotelCalifornia) [inline include/pros/imu.hpp:448]: should not be indented here

**Review by WillXuCodes — DISMISSED** [review]
LGTM

**Review by HotelCalifornia — APPROVED** [review]

**Pull request merged**